src/sql/workbench/api/common/notebooks/notebookUtils.ts

```diff
@@ -81,47 +88,52 @@ export function convertToVSCodeCellOutput(output: azdata.nb.ICellOutput): vscode
 
 export function convertToADSNotebookContents(notebookData: vscode.NotebookData): azdata.nb.INotebookContents {
 	let result = {
-		cells: notebookData.cells?.map<azdata.nb.ICellContents>(cell => {
+		cells: notebookData?.cells?.map<azdata.nb.ICellContents>(cell => {
```

Comment:
Param type should be `vscode.NotebookData` if this is actually expected to sometimes be undefined

Reply:
Fixed.